### ISSUE TITLE
Errors when running `bundle exec erblint --lint-all`

### ISSUE THREAD

**Issue by AlbertoPdRF** [thepracticaldev/dev.to#8370]
**Describe the bug**

Running `bundle exec erblint --lint-all` outputs `130 error(s) were found in ERB files`.

**To Reproduce**

1. Run `bundle exec erblint --lint-all`
2. See 130 errors

**Expected behavior**

There shouldn't be any errors.

**Screenshots**

**Desktop (please complete the following information):**

- OS: Ubuntu 20.04 (WSL2)
- Browser:
- Version:

**Smartphone (please complete the following information):**

- Device:
- OS:
- Browser:
- Version:

**Additional context**

I found this while working on #8369, as one error (fixed on that PR) was preventing me from creating a commit with changes on the file with such error. I had to fix that error manually as the pre-commit hook, which runs erblint with the `--autocorrect` flag, wouldn't work.

I'm happy to work on fixing this issue!

**Comment by rhymes** [comment]
Thanks for the issue! We'll take your request into consideration and follow up if we decide to tackle this issue.

To our amazing contributors: issues labeled `type: bug` are always up for grabs, but for feature requests, please wait until we add a `ready for dev` before starting to work on it.

To claim an issue to work on, please leave a comment. If you've claimed the issue and need help, please ping @thepracticaldev/oss and we will follow up within 3 business days.

For full info on how to contribute, please check out our [contributors guide](https://github.com/thepracticaldev/dev.to/blob/master/CONTRIBUTING.md).

**Comment by rhymes** [comment]
Time to take a second stab at this I guess :D

First part was https://github.com/thepracticaldev/dev.to/issues/1842

**Comment by AlbertoPdRF** [comment]
As that's how it was done in #1842, I'll submit a separate pull request for each of the `app/views/` folder (e.g. `app/views/listings`) to keep them small and easier to review.

**Comment by rhymes** [comment]
Thank you @AlbertoPdRF! Small PRs are the key :D

**Comment by AlbertoPdRF** [comment]
There are PRs to fix all the errors already, but there's still something I haven't fixed. If I run `bundle exec erblint --lint-all` I have the following output:

<details>
<summary>Output</summary>

```ruby
Linting 438 files with 12 linters...

Exception occured when processing: app/views/internal/listings/index.html.erb
If this file cannot be processed by erb-lint, you can exclude it in your configuration file.
Parser::Source::Range: end_pos must not be less than begin_pos
/home/apdrf/.rbenv/versions/2.7.1/lib/ruby/gems/2.7.0/gems/parser-2.7.1.3/lib/parser/source/range.rb:39:in `initialize'
/home/apdrf/.rbenv/versions/2.7.1/lib/ruby/gems/2.7.0/gems/better_html-1.0.15/lib/better_html/tokenizer/location.rb:12:in `initialize'
/home/apdrf/.rbenv/versions/2.7.1/lib/ruby/gems/2.7.0/gems/better_html-1.0.15/lib/better_html/parser.rb:204:in `new'
/home/apdrf/.rbenv/versions/2.7.1/lib/ruby/gems/2.7.0/gems/better_html-1.0.15/lib/better_html/parser.rb:204:in `build_location'
/home/apdrf/.rbenv/versions/2.7.1/lib/ruby/gems/2.7.0/gems/better_html-1.0.15/lib/better_html/parser.rb:197:in `build_node'
/home/apdrf/.rbenv/versions/2.7.1/lib/ruby/gems/2.7.0/gems/better_html-1.0.15/lib/better_html/parser.rb:171:in `build_attribute_node'
/home/apdrf/.rbenv/versions/2.7.1/lib/ruby/gems/2.7.0/gems/better_html-1.0.15/lib/better_html/parser.rb:145:in `build_tag_attributes_node'
/home/apdrf/.rbenv/versions/2.7.1/lib/ruby/gems/2.7.0/gems/better_html-1.0.15/lib/better_html/parser.rb:128:in `build_tag_node'
/home/apdrf/.rbenv/versions/2.7.1/lib/ruby/gems/2.7.0/gems/better_html-1.0.15/lib/better_html/parser.rb:75:in `build_document_node'
/home/apdrf/.rbenv/versions/2.7.1/lib/ruby/gems/2.7.0/gems/better_html-1.0.15/lib/better_html/parser.rb:45:in `ast'
/home/apdrf/.rbenv/versions/2.7.1/lib/ruby/gems/2.7.0/gems/erb_lint-0.0.33/lib/erb_lint/processed_source.rb:14:in `ast'
/home/apdrf/.rbenv/versions/2.7.1/lib/ruby/gems/2.7.0/gems/erb_lint-0.0.33/lib/erb_lint/linters/right_trim.rb:16:in `run'
/home/apdrf/.rbenv/versions/2.7.1/lib/ruby/gems/2.7.0/gems/erb_lint-0.0.33/lib/erb_lint/runner.rb:25:in `block in run'
/home/apdrf/.rbenv/versions/2.7.1/lib/ruby/gems/2.7.0/gems/erb_lint-0.0.33/lib/erb_lint/runner.rb:24:in `each'
/home/apdrf/.rbenv/versions/2.7.1/lib/ruby/gems/2.7.0/gems/erb_lint-0.0.33/lib/erb_lint/runner.rb:24:in `run'
/home/apdrf/.rbenv/versions/2.7.1/lib/ruby/gems/2.7.0/gems/erb_lint-0.0.33/lib/erb_lint/cli.rb:113:in `block in run_with_corrections'
/home/apdrf/.rbenv/versions/2.7.1/lib/ruby/gems/2.7.0/gems/erb_lint-0.0.33/lib/erb_lint/cli.rb:111:in `times'
/home/apdrf/.rbenv/versions/2.7.1/lib/ruby/gems/2.7.0/gems/erb_lint-0.0.33/lib/erb_lint/cli.rb:111:in `run_with_corrections'
/home/apdrf/.rbenv/versions/2.7.1/lib/ruby/gems/2.7.0/gems/erb_lint-0.0.33/lib/erb_lint/cli.rb:63:in `block in run'
/home/apdrf/.rbenv/versions/2.7.1/lib/ruby/gems/2.7.0/gems/erb_lint-0.0.33/lib/erb_lint/cli.rb:60:in `each'
/home/apdrf/.rbenv/versions/2.7.1/lib/ruby/gems/2.7.0/gems/erb_lint-0.0.33/lib/erb_lint/cli.rb:60:in `run'
/home/apdrf/.rbenv/versions/2.7.1/lib/ruby/gems/2.7.0/gems/erb_lint-0.0.33/exe/erblint:9:in `<top (required)>'
/home/apdrf/.rbenv/versions/2.7.1/bin/erblint:23:in `load'
/home/apdrf/.rbenv/versions/2.7.1/bin/erblint:23:in `<top (required)>'
/home/apdrf/.rbenv/versions/2.7.1/lib/ruby/site_ruby/2.7.0/bundler/cli/exec.rb:63:in `load'
/home/apdrf/.rbenv/versions/2.7.1/lib/ruby/site_ruby/2.7.0/bundler/cli/exec.rb:63:in `kernel_load'
/home/apdrf/.rbenv/versions/2.7.1/lib/ruby/site_ruby/2.7.0/bundler/cli/exec.rb:28:in `run'
/home/apdrf/.rbenv/versions/2.7.1/lib/ruby/site_ruby/2.7.0/bundler/cli.rb:476:in `exec'
/home/apdrf/.rbenv/versions/2.7.1/lib/ruby/site_ruby/2.7.0/bundler/vendor/thor/lib/thor/command.rb:27:in `run'
/home/apdrf/.rbenv/versions/2.7.1/lib/ruby/site_ruby/2.7.0/bundler/vendor/thor/lib/thor/invocation.rb:127:in `invoke_command'
/home/apdrf/.rbenv/versions/2.7.1/lib/ruby/site_ruby/2.7.0/bundler/vendor/thor/lib/thor.rb:399:in `dispatch'
/home/apdrf/.rbenv/versions/2.7.1/lib/ruby/site_ruby/2.7.0/bundler/cli.rb:30:in `dispatch'
/home/apdrf/.rbenv/versions/2.7.1/lib/ruby/site_ruby/2.7.0/bundler/vendor/thor/lib/thor/base.rb:476:in `start'
/home/apdrf/.rbenv/versions/2.7.1/lib/ruby/site_ruby/2.7.0/bundler/cli.rb:24:in `start'
/home/apdrf/.rbenv/versions/2.7.1/lib/ruby/gems/2.7.0/gems/bundler-2.1.4/exe/bundle:46:in `block in <top (required)>'
/home/apdrf/.rbenv/versions/2.7.1/lib/ruby/site_ruby/2.7.0/bundler/friendly_errors.rb:123:in `with_friendly_errors'
/home/apdrf/.rbenv/versions/2.7.1/lib/ruby/gems/2.7.0/gems/bundler-2.1.4/exe/bundle:34:in `<top (required)>'
/home/apdrf/.rbenv/versions/2.7.1/bin/bundle:23:in `load'
/home/apdrf/.rbenv/versions/2.7.1/bin/bundle:23:in `<main>'

No errors were found in ERB files
```
</details>

I don't really get what's the problem or how to fix it, any help?

**Comment by rhymes** [comment]
@AlbertoPdRF I tried with `bundle exec erblint app/views/internal/listings/index.html.erb` and used a very secretive technique: delete blocks of the source code until I figure out which line is the culprit :D 

I did find it, it's the `=` in this line:

https://github.com/thepracticaldev/dev.to/blob/2660c76fca7226256f0ce07586d2aa60d3f1ff86/app/views/internal/listings/index.html.erb#L72

see `data-action=`

**Comment by AlbertoPdRF** [comment]
Ah! Nice finding 😄 Is it safe to remove the whole `data-action=` part of the line?

**Comment by rhymes** [comment]
@AlbertoPdRF think so! I'll make sure to test it in the PR
